keras/ops/math.py

```diff
@@ -968,6 +968,37 @@ def erf(x):
     return backend.math.erf(x)
 
 
+class ErfInv(Operation):
```

Comment:
Since the function name is `erfinv`, the class name should be `Erfinv`. `ErfInv` would be for `erf_inv`.